localstack/services/ssm/provider.py

```diff
@@ -34,7 +126,15 @@ def _has_secrets(names: ParameterNameList) -> Boolean:
         return maybe_secret is not None
 
     @staticmethod
-    def _normalize_name(param_name: ParameterName) -> ParameterName:
+    def _normalize_name(param_name: ParameterName, validate=False) -> ParameterName:
+        if validate:
+            msg = (
```

Comment:
nit: could move this in the if block or directly in the constructor

Reply:
Good catch, updated 👍